Sources/ShopifyCheckout/CheckoutView.swift

```diff
@@ -145,8 +145,10 @@ extension CheckoutView: WKNavigationDelegate {
 		if isCheckout(url: response.url) && response.statusCode >= 400 {
 			CheckoutView.cache = nil
 			switch response.statusCode {
-			case 404, 410:
+			case 410:
 				viewDelegate?.checkoutViewDidFailWithError(error: .checkoutExpired(message: "Checkout has expired"))
+			case 404:
+				viewDelegate?.checkoutViewDidFailWithError(error: .sdkError(underlying: CheckoutLiquidError.unmigratedCheckoutError(message: "The checkout url provided has resulted in an error. The store is still using checkout.liquid, whereas the checkout SDK only supports checkout with extensibility.")))
```

Comment:
Is it a guarantee that 404 = unmigrated? What if the cart/checkout simply doesn't exist?

Reply:
It's just not great is it.. I'll move the needle on a more appropriate status code response over the coming days